src/pages/dashboard/[subdomain]/editor.tsx

```diff
@@ -508,6 +508,13 @@ export default function SubdomainEditor() {
     />
   )
 
+  const discardChanges = () => {
+    const draftKey = getDraftKey()
+    delStorage(draftKey)
+    // Return back to last page
+    router.back()
```

Comment:
I suggest using `queryClient.invalidateQueries` to invalidate the cached request. This should trigger `setInitialContent` and `useEditorState.setState`, refreshing the page data, so there's no need to go back to the previous page.